packages/p2p/src/pages/orders/order-table/order-table-header.scss

```diff
@@ -2,4 +2,9 @@
     display: flex;
     flex: 1;
     flex-direction: column;
+    .dc-table__header {
+        > .order-table-grid {
+            padding: 0 2.4rem;
+        }
+    }
```

Comment:
does passing className not work to the `<Table.Head>` component?

Reply:
yeah. it would have the same effect.. 
 but not to the table.header.. instead to the <Table.row>

Reply:
ah alright 👍